### ISSUE TITLE
how to load json produced with convert_to_threejs.py

### ISSUE THREAD

**Issue by grgr** [mrdoob/three.js#5983]
I tried to load json-files produced with convert_to_threejs.py with ObjectLoader, JSONLoader and SceneLoader.
No chance, the structure of the json-files does not fit the expected structure from the Loaders.

IMPORTANT: If I upload the files to clara.io and then download them in threejs-json-format everthing works as expected (with ObjectLoader). The json-file structure from clara.io is differrent than the one produced with convert_to_threejs.py.

Is this a known issue?
Are there some hints I could follow?
Is someone working on convert_to_threejs.py actually?
If not I could give it a try. The differences are not too big.

Thanks in advance

**Comment by mrdoob** [comment]
`JSONLoader` should be able to load them. What error are you getting?
`convert_to_threejs.py` is kind of deprecated though...

**Comment by grgr** [comment]
I get the console message that I should use SceneLoader. But SceneLoader is substituted with ObjectLoader in #5560. And ObjectLoader needs a slightly different json Format than the one produced with convert_to_threejs.py.
Is there another possibility to convert .fbx (and .obj) at the moment or someone working on it or some other plans?
Just wanted to be sure before I start something.

**Comment by mrdoob** [comment]
Does Blender load .fbx?

**Comment by mrdoob** [comment]
You can load the .fbx in Blender and export from there to .json using [the exporter](https://github.com/mrdoob/three.js/tree/master/utils/exporters/blender).

**Comment by grgr** [comment]
I will try if the conversion is automatable with commandline blender.
So, can I conclude there is no work going on to fit convert_to_threejs.py to the needs of ObjectLoader?

**Comment by mrdoob** [comment]
Yeah, `convert_to_threejs.py` is very outdated and unmaintained.
If you want a command line approach, maybe you can try node.js + OBJLoader.js + object.toJSON()?

**Comment by grgr** [comment]
Thank you for the hint with node.js! I will give it a try.

**Comment by mrdoob** [comment]
Let me know how it goes! :)

**Comment by bsenftner** [comment]
For what it’s worth, we use convert_to_threes.py at the www.3d-Avatar-Store.com <http://www.3d-avatar-store.com/>, and load with the JSONLoader. 

Is that path being phased out? 

Sincerely,
-Blake Senftner

**Comment by mrdoob** [comment]
Eventually, yes.

**Comment by grgr** [comment]
So @mrdoob, I did not have time before but here is something like an answer how it goes :) 

OBJLoader is great for .obj-files but loads .fbx or .3DS with Mesh.geometry.boundingSphere.radius = 0 and Mesh.geometry.boundingSphere.center has only NaN values . 
But these files are converted with values with convert_to_threejs.py. 

You gave me the hint with OBJLoader. But OBJLoader like SceneLoader can only be found in examples/js/loaders and not in src/loaders. The fact that SceneLoader is  kind of deprecated made me think all the stuff in the examples/js/loaders is kind of deprecated. So I did not want to use it. 
Finally, after your hint, I threw away my worries about deprecation and got the json-files produced with convert_to_threejs.py to show up with SceneLoader and STLLoader as  GeometryHandler :)

**Comment by mrdoob** [comment]
>  The fact that SceneLoader is kind of deprecated made me think all the stuff in the examples/js/loaders is kind of deprecated.

Oh, good point... I should create a `deprecated` folder...

**Comment by RunninglVlan** [comment]
> Yeah, convert_to_threejs.py is very outdated and unmaintained.

Hi. I just tried to load an object converted from FBX with help of convert_to_threejs.py and JSONLoader threw an Error that I need to use SceneLoader for the loading. Then I found out that it's deprecated. Normally when something becomes deprecated, it is replaced by something new. There is no mention in SceneLoader however that we need to use ObjectLoader instead.

Now I also find out that convert_to_threejs.py is outdated, but there's no mention about it in its README.md either. This got me really frustrated. I think it would be good to have a message in SceneLoader that it's replaced by ObjectLoader, and in convert_to_threejs.py that it isn't compatible with ObjectLoader yet.
